plugins/inputs/snmp/snmp_test.go

```diff
@@ -139,11 +139,10 @@ func TestFieldInit(t *testing.T) {
 	for _, txl := range translations {
 		f := Field{Oid: txl.inputOid, Name: txl.inputName, Conversion: txl.inputConversion}
 		err := f.init()
-		if !assert.NoError(t, err, "inputOid='%s' inputName='%s'", txl.inputOid, txl.inputName) {
-			continue
-		}
-		assert.Equal(t, txl.expectedOid, f.Oid, "inputOid='%s' inputName='%s' inputConversion='%s'", txl.inputOid, txl.inputName, txl.inputConversion)
-		assert.Equal(t, txl.expectedName, f.Name, "inputOid='%s' inputName='%s' inputConversion='%s'", txl.inputOid, txl.inputName, txl.inputConversion)
+		require.NoError(t, err, "inputOid='%s' inputName='%s'", txl.inputOid, txl.inputName)
```

Comment:
This will change the test behavior. Previously, the loop continued if the error was not nil, now the whole test will fail. Not sure which behavior is intended... I just want to make you aware of it, if you are not already... ;-)

Reply:
Exactly! I believe that previous approach (continue to another iteration of `for` when `err != nil`) was some kind of mistake. Why not fail anyway for that case?

(both approaches  pass ;))

Reply:
Ok, so we can keep your way. Just wanted to make sure. :-)